app/assets/stylesheets/partials/_typography.scss

```diff
@@ -35,3 +34,20 @@ p.font-xsmall{
 .hero__heading-small {
   display: block;
 }
+
+/*
+====================================
+Utility bold
+Pulled from GDS elements typography
+====================================
+*/
+.bold {
+  font-weight: 700;
+}
+label > .bold {
+  display: block; /*temp fix until form story can fix label structure*/
```

Comment:
I think I remember this causing issues in the Dec Service, what's it used for?

Reply:
I don't know the origin. I've checked whole project for usages and cannot find any. Might be, that it is some "forgotten relict from the past".
I'll remove it then.